src/aac/spec/spec.yaml

```diff
@@ -6,24 +6,60 @@ data:
   fields:
     - name: import
       type: string[]
+      description: |
+        A list references to dependent models whose definitions should be
+        available for use by the current model.
     - name: enum
       type: enum
+      description: |
+        A definition that represents an enumerated type of value.
+
+        An example of when to use an `enum' is when you want to provide several
+        different options for a single value.
     - name: data
       type: data
+      description: |
+        A definition that represents a data value.
+
+        An example of when to use a `data' definition is when you want to define
+        some part of the system that can be an input and/or output to another
+        part of the system.
```

Comment:
I don't think this is quite accurate as `data` is _the_ keystone to our DSL and represents a general structure that acts as the backbone for the rest of the DSL. All of that to say that data is probably not the correct name for this key; we should have a discussion around that. 

For now, I'd make this more nebulous or maybe rather than focus on data as information, focus on data as shorthand for datastructure.